CHANGELOG.md

```diff
@@ -7,6 +7,8 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased](https://github.com/open-telemetry/opentelemetry-python/compare/v1.11.1-0.30b1...HEAD)
 
+- Fix LoggingHandler to handle LogRecord with exc_info=False
+  ([#2689](https://github.com/open-telemetry/opentelemetry-python/pull/2689))
```

Comment:
```suggestion
  ([#2690](https://github.com/open-telemetry/opentelemetry-python/pull/2690))
```

Reply:
oops thanks for catching this, thought it was issue number